tests/engine/reliability/test_deadlocks.py

```diff
@@ -22,7 +22,7 @@ def run(n):
     run(500)
 
 
-@pytest.mark.skip(reason="Causes a deadlock.")
+@pytest.mark.skip(reason="This test takes multiple minutes")
 def test_loop_wait_for_many_tasks():
```

Comment:
This actually may still cause a deadlock locally too. I'm not sure why yet but this should still be an improvement.